x-pack/plugins/osquery/cypress/e2e/tiers/endpoint_complete.cy.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { checkOsqueryResponseActionsPermissions } from '../../tasks/response_actions';
+import { ServerlessRoleName } from '../../../../../test_serverless/shared/lib';
```

Comment:
I believe the serverless cypress tests are failing because this (and the other tests in `/e2e/tiers`) should be `import { ServerlessRoleName } from '../../support/roles';`

Reply:
yup I reverted the change